modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/global_settings.jspf

```diff
@@ -63,6 +65,57 @@
 	</clay:sheet-section>
 </div>
 
+<div id="<portlet:namespace />customProductionName">
+	<clay:sheet-section>
+		<h3 class="sheet-subtitle"><liferay-ui:message key="custom-production-name" /></h3>
+
+		<clay:alert
+			displayType="info"
+			message="using-a-custom-production-name-requires-you-to-set-your-chosen-replacement-term-in-the-publications-instance-settings-page-and-to-optionally-create-translations-for-the-replacement-term-in-the-language-override-portlet-these-configuration-pages-are-linked-below"
+		/>
+
+		<div class="sheet-text"><liferay-ui:message key="replaces-usages-of-the-term-production-with-a-custom-name" /></div>
+
+		<div class="form-group">
+			<aui:button
+				href='<%=
+					PortletURLBuilder.create(
+						PortalUtil.getControlPanelPortletURL(request, ConfigurationAdminPortletKeys.INSTANCE_SETTINGS, PortletRequest.RENDER_PHASE)
+					).setMVCRenderCommandName(
+						"/configuration_admin/edit_configuration"
+					).setRedirect(
+						ParamUtil.getString(request, "backURL", currentURL)
+					).setParameter(
+						"factoryPid", CTConfiguration.class.getName()
+					).setParameter(
+						"pid", CTConfiguration.class.getName()
+					).buildString()
+				%>'
+				icon="icon-new-window"
+				name="publicationsInstanceSettingsLink"
+				target="_blank"
+				value='<%= LanguageUtil.get(request, "go-to-publications-instance-settings") %>'
+			>
+			</aui:button>
+
+			<aui:button
+				href='<%=
+					PortletURLBuilder.create(
+						PortalUtil.getControlPanelPortletURL(request, "com_liferay_portal_language_override_web_internal_portlet_PLOPortlet", PortletRequest.RENDER_PHASE)
+					).setRedirect(
+						ParamUtil.getString(request, "backURL", currentURL)
+					).buildString()
+				%>'
+				icon="icon-new-window"
+				name="languagePortletLink"
+				target="_blank"
+				value='<%= LanguageUtil.get(request, "go-to-language-override-portlet") %>'
+			>
+			</aui:button>
```

Comment:
I think we can remove the link to the Language Override portlet. It's not obvious which language keys need to be override and will probably create more confusion to the customers. Customers can provide their own translations for the replacement term in Instance Settings. 

We need to update this lang key because it's doesn't work this way.
`
using-a-custom-production-name-requires-you-to-set-your-chosen-replacement-term-in-the-publications-instance-settings-page-and-to-optionally-create-translations-for-the-replacement-term-in-the-language-override-portlet-these-configuration-pages-are-linked-below
`

@david-truong what do you think?

Reply:
I didn't realize we could do it all through instance settings.  I think that can work...